--- a/Makefile	2016-01-01 18:51:57.000000000 +0100
+++ b/Makefile	2016-03-08 10:07:13.529974400 +0100
@@ -5,10 +5,10 @@
 all: osd
 
 /tmp/%.o: %.c
-	gcc -c -o $@ $< $(CPPFLAGS)
+	$(CC) -c -o $@ $< $(CPPFLAGS)
 
 osd: /tmp/main.o /tmp/render.o /tmp/telemetry.o /tmp/frsky.o /tmp/ltm.o /tmp/mavlink.o /tmp/smartport.o
-	gcc -o /tmp/$@ $^ $(LDFLAGS)
+	$(CC) -o /tmp/$@ $^ $(LDFLAGS)
 
 clean:
 	rm -f /tmp/osd /tmp/*.o /tmp/*~
