--- a/Makefile	2016-01-01 18:51:57.000000000 +0100
+++ b/Makefile	2016-03-08 10:07:13.529974400 +0100
@@ -7,19 +7,19 @@
 
 
 %.o: %.c
-	gcc -c -o $@ $< $(CPPFLAGS)
+	$(CC) -c -o $@ $< $(CPPFLAGS)
 
 
 rx: rx.o lib.o radiotap.o fec.o
-	gcc -o $@ $^ $(LDFLAGS)
+	$(CC) -o $@ $^ $(LDFLAGS)
 
 
 tx: tx.o lib.o fec.o
-	gcc -o $@ $^ $(LDFLAGS)
+	$(CC) -o $@ $^ $(LDFLAGS)
 
 
 rx_status_test: rx_status_test.o
-	gcc -o $@ $^ $(LDFLAGS)
+	$(CC) -o $@ $^ $(LDFLAGS)
 
 clean:
 	rm -f rx tx *~ *.o
